### PR TITLE
feat(TMRS-268): update queries and article list component to use upda…

### PR DESCRIPTION
**Pull request by douglasmik**
feat(TMRS-268): update queries and article list component to use updated url field, which uses mapped `categoryPath` value
[TMRS-268](https://nidigitalsolutions.jira.com/browse/TMRS-268)

### Checklist

- [ ] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):

Include screenshots if needed.

[TMRS-268]: https://nidigitalsolutions.jira.com/browse/TMRS-268?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/article-list/src/article-list-item.js [hunk 1]
```diff
@@ -107,32 +107,28 @@ const ArticleListItem = props => {
...
-                isLcpItem={index === 0}
-              >
-                <Animations.FadeIn>{children}</Animations.FadeIn>
-              </Card>
-            </ListItemWrapper>
-          </Link>
-        );
-      }}
+      {() => (
+        <Link url={props.article.url}>
```
Comment (yukiinasawa) [inline packages/article-list/src/article-list-item.js:111]: Isn't it `props.article.categoryPath` ?

File: packages/fixture-generator/src/types.ts [hunk 2]
```diff
@@ -1351,6 +1351,8 @@ export interface Article extends ArticleInterface {
   /** Topics that the requested article belong to */
   topics?: (Topic | null)[] | null;
 
+  categoryPath?: Url | null;
```
Comment (yukiinasawa) [inline packages/fixture-generator/src/types.ts:1354]: Isn't it `categoryPath: Url` as per prev our conversation earlier?

**Review by HarisHodzicnt — APPROVED** [review]

File: packages/article-list/src/article-list-item.js (hunk 1 above)
Comment (douglasmik) [reply packages/article-list/src/article-list-item.js:111]: I've mapped the `categoryPath` field to the `url` in the query;
e.g. `url: categoryPath`

File: packages/fixture-generator/src/types.ts (hunk 2 above)
Comment (douglasmik) [reply packages/fixture-generator/src/types.ts:1354]: followed type for `url` where this is for the fixture generator, that is used to mock different components, etc - they all have conditional `null` values

File: packages/article-list/src/article-list-item.js (hunk 1 above)
Comment (yukiinasawa) [reply packages/article-list/src/article-list-item.js:111]: If i understand correctly - we receive `categoryPath` as `url` so then we don't need to add `categoryPath` fixtures you've added into fixtures files?

Comment (douglasmik) [reply packages/article-list/src/article-list-item.js:111]: fixtures seemed to need it, as they were not 'querying' for the data but working from a static object

**Review by yukiinasawa — APPROVED** [review]

**Pull request merged**